libraries/provision/ansible/playbooks/tasks/fetch-sync-gateway-logs.yml

```diff
@@ -69,4 +69,4 @@
   become: yes
   # Do not validate checksum as this is just a passive grab of the logs. Since they are continuously being appended to, the checksum may fail
   fetch: src=/home/sync_gateway/logs/sg_warn.log dest=/tmp/sg_logs/{{ inventory_hostname }}/ fail_on_missing=yes flat=yes validate_checksum=no
-  when: sw.stat.exists
```

Comment:
Seems to be the removal of the blank line at the end of the file from moving the trace logs fix to another branch, will fix so that the change is not needed